src/redux/configureStore.js

```diff
@@ -1,12 +1,15 @@
 import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
- [ ] I got this error when I'm trying to start the app. 

![Capture](https://user-images.githubusercontent.com/64173189/150462648-9e408559-e5b0-49d2-9fde-2dba4cd8a68f.JPG)

Just import compose to get rid of this error. 

```suggestion
import { createStore, combineReducers, applyMiddleware, compose } from 'redux';
```

Reply:
Awesome, thank you @BrianSammit 🙏 ! I implemented the suggested change and will gladly merge this pull request now 😉. Have a happy weekend!